src/router/images/spec.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * A Upload Image form
+ * @typedef {object} UploadImageForm
+ * @property {string[]} image - Binary files to upload - Binary
+ */
+
+/**
+ * POST /images/
+ * @tags Images
+ * @summary Upload single or multiple images and return image info.
+ * @param {UploadImageForm} request.body.required - Request - multipart/form-data
+ * @return {object} 200 - Returns image info
+ * @return {object} 400 - Returns error message for no image in request
+ * @return {object} 403 - Returns error message for forbidden request
+ * @return {object} 500 - Returns error message for internal server error
+ * @example response - 200 - UPLOAD_SUCCESS
+ * {
+ *  "msg": "Image Saved Successfully",
+ *  "imageInfo": [
+ *       {
+ *         "id": "fad6c4b32695ff7e0a35f27fd42ba7b9",
+ *         "path": "src\\res\\image\\fad6c4b32695ff7e0a35f27fd42ba7b9",
```

Comment:
path should be an url available to client-side
ex) server.kuaaa.net/api/image/fad6c4b32695ff7e0a35f27fd42ba7b9